Added steps on sending in-app events to Segment [DOC-1131] (#7683)
segmentio/segment-docs: src/connections/destinations/catalog/adjust/index.md

If you're using Adjust's iOS SDK, it will automatically takes care of duplicate purchase events. Segment uses Adjust's default deduplication (using `transactionId`) when you send an `orderId` (see the [ecommerce spec](/docs/connections/spec/ecommerce/v2/#order-completed)).

### Sending in-app events from Adjust to Segment

> Customers can forward in-app events from Adjust to Segment by following the steps below:

Reviewer: ```suggestion
Customers can forward in-app events from Adjust to Segment by following these steps:
```
